GG/src/SDL/SDLGUI.cpp

```diff
@@ -380,9 +380,8 @@ namespace {
                 case GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT_EXT:
                     return "One of the framebuffer attachments is incomplete.";
                 default:
-                    std::stringstream ss;
-                    ss << "Framebuffer creation failed. Status: " << m_status;
-                    return ss.str().c_str();
+                    std::string * s= new std::string("Framebuffer creation failed. Status: " + std::to_string(m_status));
+                    return s->c_str();
```

Comment:
This will leak memory each time it's executed, as you create a new string and don't delete it.

Reply:
Again, instead of trying to create and return a pointer to a local that goes out of scope, just handle all the possible, or at least some more of the possible, values of `m_status`. See https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glCheckFramebufferStatus.xhtml